pacehrh/R/pace_config.R

```diff
@@ -167,3 +167,38 @@ SetGlobalStartEndYears <- function(start = 2020, end = 2040, shoulderYears = 1)
 
   return(invisible(NULL))
 }
+
+#' Set Rounding Law
+#'
+#' The rounding law determines how the system handles rounding for
+#' processes that in nature would always return integer counts, such as
+#' births.
+#'
+#' For example: Take 35 age groups, each containing 100 fertile women, and
```

Comment:
is the early / late concept only applies to the fertility rate? For other calculation it seems they don't matter (only none option matters) I feel that this description should clarify that also mentioned that the default is "early" if using NULL